lib/unifex/specs.ex

```diff
@@ -81,15 +81,40 @@ defmodule Unifex.Specs do
 
     functions_docs = parse_docs(config, specs_file)
 
+    dirty_functions =
+      config
+      |> Keyword.get_values(:dirty_functions)
+      |> List.flatten()
+      |> Enum.flat_map(fn {dirty_func, type} ->
+        dirty_name =
+          case dirty_func do
+            {name, _arity} -> name
+            name -> name
+          end
+
+        {matched_name, matched_args} = List.keyfind(functions_args, dirty_name, 0, {nil, nil})
+
+        if matched_name != nil and matched_args != nil and
+             dirty_func in [matched_name, {matched_name, length(matched_args)}] do
```

Comment:
these nil checks are required, otherwise if `List.keyfind()` doesn't match `length(matched_args) ` results in a crash